Automattic/sensei
# Add sensei logo to the default learning mode template

Resolves #6742 

## Proposed Changes
* Create a sensei logo block with a link to sensei
* Add the block to the default learning mode template.
* Update the Sensei->assets->get_icon to receive width and height

## Testing Instructions

Figma: 8mY8ui4zYK1Ylw2yIFi9Lx-fi-20020517_40169

1. Go to this branch
2. Reset all stored changes on the learning mode template (default)
3. Check if the sensei logo is visible on the sidebar
4. Click on the link and check if it is correct.

### Supported Themes
It should work on
* Course Theme
* Divi Theme
* Astra Theme.

### Screenshots

##### Editor
![image](https://github.com/Automattic/sensei/assets/38718/ac47d566-5145-4f10-8929-6d35b9140f53)


#### Frontend
![image](https://github.com/Automattic/sensei/assets/38718/c179f9ae-4a6e-453f-b110-5fc5ff9394d9)



Both the PR author and reviewer are responsible for ensuring the checklist is completed. -->
